packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -110,6 +110,27 @@ module.exports = {
       }
     ],
     loaders: [
+      // Default loader: load all assets from appSrc that are not already
+      // handled by other loaders with the url loader.
+      // Note: Any changes to the other loader tests that apply to appSrc
+      // need to be reflected here too.
+      // E.g., when adding a loader for new supported file extensions
+      // in appSrc, we need to add the new extension to this list too.
+      //
+      // "file" loader makes sure those assets get served by WebpackDevServer.
+      // When you `import` an asset, you get its (virtual) filename.
+      // In production, they would get copied to the `build` folder.
+      // "url" loader works like "file" loader except that it embeds assets
+      // smaller than specified limit in bytes as data URLs to avoid requests.
+      {
+        test: /\.(?!(js|jsx|css|json|html)(\?.*)?$)[^\.]+$/,
```

Comment:
```
exclude: /\.(js|jsx|css|json)$/
```
would work the same way, but then it's no longer the same regex, so it's harder to notice if we introduce discrepancies between them.